.github/workflows/pull_request_title.yml

```
@@ -0,0 +1,21 @@
+name: Gitmoji + Commitlint Pull Request Title Rule 
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Both actions have newer versions available. Latest and greatest! 🚀 

```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
```

---

Ah, that's awesome.. I didn't know there was already v3.